components/BuyingGuide.tsx

```diff
@@ -0,0 +1,44 @@
+import Image from "next/image";
+
+type Props = {
+  glo: number;
+  closeModal: any;
+};
+
+export default function BuyingGuide({ glo, closeModal }: Props) {
+  return (
+    <>
+      <div className="flex flex-col bg-pine-50 text-pine-900 pt-[28px] px-[23px] pb-[20px]">
+        <div className="flex flex-row justify-between">
+          <div></div>
+          <button className="" onClick={() => closeModal()}>
+            <Image alt="x" src="/x.svg" height={16} width={16} />
+          </button>
+        </div>
+        <div className="text-4xl text-[42px]">Buying Glo</div>
+        <div className="text-2xl">A step by step guide</div>
+      </div>
+      <div className="flex flex-col space-y-4 text-pine-900 mt-3 mb-5 mx-4 text-sm">
+        <div>
+          As Glo is in beta you can only get Glo by exchanging it with USDC
+          using an app called Uniswap. This process consists of 12 steps:
+        </div>
+
+        <ol className="list-decimal mx-4">
+          <li>Create an account with Coinbase</li>
+          <li>Buy {glo + 1} USDC on Coinbase via wire transfer</li>
+          <li>Send {glo + 1} USDC to: 0x...ef81 🔗 on Polygon</li>
```

Comment:
wallet address should be dynamically populated :p

Reply:
fixed